### DISCUSSION TITLE
How can I make imports from the main to the renderer? or from common package inside the project?

### DISCUSSION THREAD

**Discussion by MiniSuperDev** [alex8088/electron-vite#297] [Q&A]
For example to use https://www.electron-trpc.dev/
I need to import some types to import the router to the main, other to the preload, and the types to the renderer.

Thanks

**Comment by alex8088** [comment]
Example: https://github.com/alex8088/EvPlayer

**Reply by itsjoeoui** [reply]
Hey @alex8088 ! I have a followup question for this. I saw that you are not importing anything in the `common` directory. But when I am setting up `electron-trpc`, I have a separate `common` directory, which will import stuff from main. and `renderer` will import stuff from `common`. But I could not get it to work properly. 

I will appreciate very much if you can show an example of electron-vite working together with electron-trpc. Thanks!
